index.js

```diff
@@ -108,9 +108,13 @@ export const Link = (props) => {
       )
         return;
 
+      onClick && onClick(event);
+      if (event.defaultPrevented) {
+        return;
+      }
```

Comment:
Maybe we save a few bytes by rewriting it as:

```suggestion
      if (!event.defaultPrevented) {
        event.preventDefault();
        navRef.current()
      }
```

?

Reply:
Make sense, why not.